app/models/petition.rb

```diff
@@ -0,0 +1,10 @@
+class Petition < ApplicationRecord
+  belongs_to :user
+  belongs_to :organization
+
+  STATUS = {
```

Comment:
Maybe we can use `enum`s here https://api.rubyonrails.org/v6.1/classes/ActiveRecord/Enum.html?